modules/cli/pages/custom-certs.adoc

```diff
@@ -0,0 +1,143 @@
+// Type: procedure
+[id="installing-cli"] 
+= Creating sites using a custom certificate authority on OpenShift
+
+By default, {skupper-name} creates certificates to establish links between sites using mTLS.
```

Comment:
there is a typo in "mTLS"

Reply:
expanded to 'mutual TLS' as per website

Reply:
Oops, my bad, I did not know the meaning of `mTLS`; maybe the extended version is better for people that are not familiar with it like me! 😅